test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+
+install_kfctl(){
+    #install kfctl cli
+    KFCTL_VERSION="v1.0.2"
+    S3_BUCKET="kfctl-binary"
+
+    if ! command -v kfctl &> /dev/null
+    then
+        aws s3 cp s3://${S3_BUCKET}/kfctl_${KFCTL_VERSION}-1-g93e95e1_linux.tar.gz /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz
```

Comment:
Where is this bucket? Please sync up with @saimidu for the correct public account.

Reply:
@nskool Do you want the implementation to change, here, or is the `aws s3 cp` command okay as it is now?

Reply:
I'd prefer the binary to be public in the account that holds public artifacts, so that it's easier to test outside of CI/CD system, although this is a custom config.

Reply:
Ack. Will make the changes in following pr.